app/Http/Requests/SaveIndividualRolesRequest.php

```diff
@@ -41,6 +41,8 @@ public function messages(): array
     {
         return [
             'roles.required' => __('You must select what you would like to do on the website.'),
+            'roles.array' => __('You must select what you would like to do on the website.'),
+            'roles.*.Illuminate\Validation\Rules\Enum' => __('You must select a valid role to perform on the website.'),
```

Comment:
I've never seen this before— can you tell me a bit more about this rule type?

Reply:
That's for when you need to set the message for an enum validation rule like `roles: [new Enum(IndividualRole::class)]`. It checks that the value is in the enum.